server/routes/forms.ts

```diff
@@ -0,0 +1,58 @@
+import { type RequestHandler, Router } from 'express'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import type { Services } from '../services'
+import { GithubRepoRequestRequest } from '../data/strapiApiTypes'
+
+export default function routes({ serviceCatalogueService, dataFilterService }: Services): Router {
+  const router = Router()
+
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  const post = (path: string, handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
+
+  get('/github-repo-request-form', async (req, res) => {
+    const [productList] = await dataFilterService.getDropDownLists({
+      productName: '',
+      useFormattedName: true,
+    })
+
+    return res.render('pages/githubRepoRequestForm', {
+      title: 'Github Repository Requst Form',
+      productList,
+    })
+  })
+
+  post('/github-repo-request-form', async (req, res): Promise<void> => {
+    const formData = req.body
+    const requestFormData = buildFormData(formData)
+    await serviceCatalogueService.postGithubRepoRequest(requestFormData)
+
+    // disable submit page, give confirmation and stay on page or redirect to new page with summary of requested data pushed to strapi
+    return res.redirect('/forms/github-repo-request-form')
+  })
+
+  return router
+}
+
+export const buildFormData = (formData: Record<string, unknown>): GithubRepoRequestRequest => {
```

Comment:
As previously mentioned, no need for the `export` as it's only being used in this file and you're not pulling it in elsewhere. Not sure it's 100% needed as it's just creating an object.